lib/manager/components/ProjectSettingsForm.js

```diff
@@ -300,19 +305,17 @@ export default class ProjectSettingsForm extends Component<Props, State> {
                 </Checkbox>
                 {autoFetchChecked
                   ? <DateTimeField
-                    dateTime={(
-                      typeof autoFetchMinute === 'number' &&
-                      typeof autoFetchHour === 'number'
-                    )
-                      ? +moment().startOf('day')
-                        .add(autoFetchHour, 'hours')
-                        .add(autoFetchMinute, 'minutes')
-                      : DEFAULT_FETCH_TIME
-                    }
+                    dateTime={typeof autoFetchMinute === 'number' &&
+                typeof autoFetchHour === 'number'
```

Comment:
The indent here is strange. Did prettier do that?

Reply:
Not sure what was going on! It's such a weird block -- the eslint formatting rules contradict each other so the only way to get them both to be happy is to have this strange middle ground. I've tried to clean it up in d54487d0